spec/controllers/admin/membership_submissions_controller_spec.rb

```diff
@@ -9,7 +9,89 @@
     end
   end
 
-  pending "GET #pending"
-  pending "POST #pending"
-  pending "POST #reject"
+  describe "GET #pending" do
+    describe "when pending submissions are zero" do
+      before do
+        allow(@controller).to receive(:require_admin!).and_return(true)
+        get :pending
+      end
+
+      it "redirects to admin_path" do
+        expect(response).to redirect_to(admin_path)
+      end
+
+      it "sets flash/info" do
+        expect(flash[:info]).to match(/submissions/i)
+      end
+    end
+
+    describe "when pending submissions are grater than zero" do
+      before do
+        @slack_membership_1 = create(:slack_membership_submission, email: 'johndoe@email.com')
+        @slack_membership_2 = create(:slack_membership_submission, email: 'johndoe2@email.com')
```

Comment:
No need to create the second membership as one is sufficient to trigger the condition. Save a few milliseconds :)

I'm not sure I have a strong preference, but I also prefer `let` and `let!` vs instance methods. I dunno if we want to standardize on that or not.